src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java

```diff
@@ -95,7 +97,7 @@ public ValidationStatusResponse validate(
             request.getHeader(ApiSdkManager.getEricPassthroughTokenHeader());
 
         Optional<OfficerFiling> maybeOfficerFiling = officerFilingService.get(filingResourceId, transaction.getId());
-        if(!isAp01Enabled){
+        if(!isAp01Enabled && !isCh01Enabled){
```

Comment:
Should this be or check instead? I see the default value for the ch01 flag is false. 
or `else` is going to take care of ap01 true and ch01 scenario?

Reply:
I'm just going to remove this code block as I don't think it's required any more